client/src/components/InterviewQuestionDetails.js

```diff
@@ -1,7 +1,6 @@
 import React, { useState } from "react";
-import { InterviewDetailButton } from '../components/Buttons'
 import { useStyles } from '../themes/theme';
-import { AnswerButton } from './Buttons'
+import { CustomButton, InterviewDetailButton } from './Buttons'
```

Comment:
is interviewDetailButton used here?

Reply:
Yes this is the interviewDetailButton. It's a little more dynamic than some of the other buttons used.

<img width="438" alt="Screen Shot 2020-11-27 at 12 24 47 PM" src="https://user-images.githubusercontent.com/61174647/100473086-8c850200-30ab-11eb-9b61-0ac9c5a64f4c.png">